core/src/migrations/000067-addConfigFilePath.ts

```diff
@@ -0,0 +1,29 @@
+const tables = [
```

Comment:
At first I didn't really want to put these references in the database. But the more I worked through the problem, it made sense to me to be there. It felt nice to have this reference to give the instance a one-to-one tie with its config source.

In production we won't use this. However, with [this story](https://www.pivotaltracker.com/story/show/178143612), we'll have a reference to the files even when they are loaded from config. That might actually help with some debugging or some future feature in either enterprise or community.